### DISCUSSION TITLE
How to map "alt+bar"?

### DISCUSSION THREAD

**Discussion by otakutyrant** [kovidgoyal/kitty#6401] [Q&A] [answered]
`map alt+bar goto_layout horizontal` does not work and I noticed [Key codes](https://sw.kovidgoyal.net/kitty/keyboard-protocol/#key-codes). But I cannot understand what the document says...

**Comment by kovidgoyal** [comment]
I have no idea what bar is. If you mean the spacebar, use space

On Wed, Jun 28, 2023 at 06:21:56AM -0700, otakutyrant wrote:
> `map alt+bar goto_layout horizontal` does not work and I noticed [Key codes](https://sw.kovidgoyal.net/kitty/keyboard-protocol/#key-codes). But I cannot understand what the document says...
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6401
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by otakutyrant** [reply]
It is `|`. `xev` shows that its name is bar.

**Comment by kovidgoyal** [comment, marked as answer]
use |

On Wed, Jun 28, 2023 at 07:08:48AM -0700, otakutyrant wrote:
> It is `|`. `xev` shows that its name is bar.
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6401#discussioncomment-6305450
> You are receiving this because you commented.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by otakutyrant** [reply]
Emmm, I do not know why "minus" is available and "bar" not. `|` works anyway. Thank you!

**Comment by kovidgoyal** [comment]
Or if you are ona  keyboard where it is not a standalone key use
shift+\ 
See the output of kitty --debug-input

On Wed, Jun 28, 2023 at 07:08:48AM -0700, otakutyrant wrote:
> It is `|`. `xev` shows that its name is bar.
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6401#discussioncomment-6305450
> You are receiving this because you commented.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
